.azure-devops/templates/run-all-tests.yml

```diff
@@ -0,0 +1,8 @@
+steps:
+- task: AzureCLI@2
+  displayName: 'Run all tests'
+  inputs:
+    azureSubscription: AzIoTCLIService
+    scriptType: bash
+    scriptLocation: inlineScript
+    inlineScript: pytest --junitxml "TEST-results.xml"
```

Comment:
Would be nice to include code coverage in the pipeline now with unit + IT's running